package.json

```diff
@@ -30,24 +31,24 @@
   },
   "homepage": "https://github.com/dabapps/eslint-config-dabapps",
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.19.0",
+    "@typescript-eslint/parser": "^2.19.0",
+    "babel-eslint": "^10.0.3",
+    "bufferutil": "^4.0.1",
+    "canvas": "^2.6.1",
     "eslint": "^6.8.0",
-    "eslint-config-prettier": "6.9.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-jest": "^23.6.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "prettier": "^1.19.1"
+    "eslint-plugin-react": "^7.18.3",
+    "eslint-plugin-react-hooks": "^2.3.0",
+    "eslint-plugin-react-native": "^3.8.1",
+    "jsonlint": "^1.6.3",
```

Comment:
why did this move to the normal dependencies?

Reply:
When we install this library as a dav-dependency and run the linter, it does not find jsonlint so I had to install it as well. To avoid it, I moved this to under regular dependencies. There may be a better way of doing this, let me know?

Reply:
This should remain as a dev dependency.

Reply:
oh, yes, moved the jsonlint back to dev.